src/components/BookList/index.js

```diff
@@ -4,6 +4,16 @@ import Col from 'react-bootstrap/Col'
 import Row from 'react-bootstrap/Row'
 
 import Book from '../Book'
+// Import Font Awesome
```

Comment:
Do these need to be add here with the Book container ?

Reply:
Again, my fault, forgot to go back to it after mergin. Will take it out.